tfgrid3/umbrel/scripts/umbrel-start.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+set -exo pipefail
+UMBREL_ROOT="${UMBREL_DISK}/umbrel"
+UMBREL_LOGS="${UMBREL_ROOT}/logs"
+USER_FILE="${UMBREL_ROOT}/db/user.json"
+
+
+REBOOT="${UMBREL_ROOT}/events/signals/reboot"
+if [ -f "$REBOOT" ] &&  grep -Fxq "true" "$REBOOT"
+ then
+    sed -i "\$d" "$REBOOT"
+    echo rebooting
+    $UMBREL_ROOT/scripts/stop;
+ else
+    echo starting
+fi
+
+
+# Configure Umbrel if it isn't already configured
+if [[ ! -f "${UMBREL_ROOT}/statuses/configured" ]]; then
+  NGINX_PORT=${NGINX_PORT:-80} NETWORK="${NETWORK:-mainnet}" "${UMBREL_ROOT}/scripts/configure"
+fi
+# make sure that we use the correct nginx configuration
+cp /templates/nginx-override.conf ${UMBREL_INSTALL_PATH}/templates/nginx-sample.conf
+REMOTE_TOR_ACCESS="false"
+if [[ -f "${USER_FILE}" ]]; then
+  REMOTE_TOR_ACCESS=$(cat "${USER_FILE}" | jq 'has("remoteTorAccess") and .remoteTorAccess')
+fi
+
+echo
+echo "======================================"
+echo "============= STARTING ==============="
+echo "============== UMBREL ================"
+echo "======================================"
+echo
+
+echo "Setting environment variables..."
+echo
+
+export IS_UMBREL_OS="false"
+
+# Increase default Docker and Compose timeouts to 240s
+# as bitcoin can take a long while to respond
+export DOCKER_CLIENT_TIMEOUT=240
+export COMPOSE_HTTP_TIMEOUT=240
+
+cd "$UMBREL_ROOT"
+
+echo "Starting karen..."
+echo
+./karen &>> "${UMBREL_LOGS}/karen.log" &
+
+echo "Starting status monitors..."
+pkill -f ./scripts/status-monitor || true
+./scripts/status-monitor memory 60 &>> "${UMBREL_LOGS}/status-monitor.log" &
+./scripts/status-monitor storage 60 &>> "${UMBREL_LOGS}/status-monitor.log" &
+./scripts/status-monitor temperature 15 &>> "${UMBREL_LOGS}/status-monitor.log" &
+./scripts/status-monitor uptime 15 &>> "${UMBREL_LOGS}/status-monitor.log" &
+
+echo "Starting memory monitor..."
+echo
+./scripts/memory-monitor &>> "${UMBREL_LOGS}/memory-monitor.log" &
+
+echo "Starting backup monitor..."
+echo
+./scripts/backup/monitor &>> "${UMBREL_LOGS}/backup-monitor.log" &
+
+echo "Starting decoy backup trigger..."
+echo
+./scripts/backup/decoy-trigger &>> "${UMBREL_LOGS}/backup-decoy-trigger.log" &
+
+compose_files=()
+
+if [[ "${REMOTE_TOR_ACCESS}" == "true" ]]; then
+  compose_files+=( "--file" "docker-compose.tor.yml" )
+fi
+
+compose_files+=( "--file" "docker-compose.yml" )
+
+echo
+echo "Starting Docker services..."
+echo
+docker-compose "${compose_files[@]}" up --detach --build --remove-orphans || {
+  echo "Failed to start containers"
+  exit 1
+}
+echo
```

Comment:
as @sameh-farouk said in https://github.com/threefoldtech/tf-images/pull/120#discussion_r1111194333
> Docker-compose v2 is now available, stable, and should be compatible with docker-compose. the only difference is that it no longer calls `docker-compose` but `docker compose`. if no specific reason to use docker-compose v1, I would suggest using the latest version.

Reply:
I'm using docker-compose v2.16.0, I can't only use the docker-compose plugin as umbrel uses `docker-compose` command widely and they are check for it's exist in the [start](https://github.com/getumbrel/umbrel/blob/d7c9f4d16317d52eef2193458935dc673790f4db/scripts/start#L23) script